package.json

```diff
@@ -13,10 +13,11 @@
     "oas-linter": "^3.0.2",
     "oas-resolver": "^2.2.7",
     "oas-validator": "^3.3.3",
-    "redoc-cli": "^0.9.6",
+    "redoc-cli": "^0.9.7",
     "remote-origin-url": "^2.0.0",
     "shelljs": "^0.7.0",
     "@redocly/openapi-cli": "0.8.4",
+    "swagger-repo": "^2.0.0-rc.15",
```

Comment:
I feel like this might not need to be a dependency anymore, with `@redocly/openapi-cli`... but need to verify. Just adding a note to remind myself later.